src/app/components/event-card/event-card.component.scss

```diff
@@ -2,58 +2,90 @@
 @import '../../mixins.scss';
 
 .event-card {
-    width: 100%;
-    height: 100%;
-    border: 3px solid variables.$yonder-light-blue;
+    width: 340px;
+    height: 255px;
+    padding: 0;
     box-shadow: 4px 4px 3px #d9d9d9;
-    border-radius: 10px;
+    border-radius: 6px 6px 6px 6px;
 }
 
-.event-image {
+.image-div {
+    justify-content: center;
+    display: flex;
     width: 100%;
+    height: 160px;
+    overflow: hidden;
+    border-radius: 6px 6px 0px 0px;
+}
+
+.event-image {
+    width: 340px;
+    height: 180px;
+    object-fit: cover;
+    object-position: 100% 0;
 }
 
 .about {
     @include flex-container(column, flex-start);
     justify-content: space-between;
+    display: flex;
+    padding: 0px 8px 8px 8px;
     margin-left: 5px;
     margin-right: 5px;
 }
 
 .first-row {
     width: 100%;
-    @include flex-container(row, flex-start);
+    margin: auto;
+    display: flex;
+    @include flex-container(row);
     margin-top: 5px;
-    margin-bottom: 10px;
     justify-content: space-between;
+    align-items: center;
 }
 
 .title {
-    font-size: 1.6vh;
+    font-size: 20px;
     font-weight: 700;
+    line-height: 110%;
 }
 
 .countdown {
-    @include flex-container(row, flex-start);
+    @include flex-container(row);
+    margin-top: 5px;
+    flex-direction: row;
+    justify-content: center;
```

Comment:
@mixin flex-container($flex-direction-value, $justify-content-value: center) {
    display: flex;
    flex-direction: $flex-direction-value;
    justify-content: $justify-content-value;
}

The default value is already center